pages/events/index.vue

```diff
@@ -124,6 +121,7 @@ import {
 
 import rawPastEvents from "~/content/events/past-community-events.json";
 import rawUpcomingEvents from "~/content/events/upcoming-community-events.json";
+import { sortEvents } from "~/utils/event-schemaorg";
```

Comment:
Same comment as in the **components/EventsFallFest/EventsFallFestUniversityDirectory.vue** file:

> Exports in the `utils` directory are globally available, so we don't need to import it.
> 
> With that in mind, I believe that it's a good idea to have very descriptive names for these functions and files.
> 
> Since this particular function sorts all kind of events, I think that it's indeed a good idea to call it `sortEvents`, but I also think that it should be in its own file (something like **utils/sortEvents.ts** and not included in a file related to the Schema.org stuff.